cms/static/js/views/modals/base_modal.js

```diff
@@ -34,6 +34,7 @@ define(["jquery", "underscore", "gettext", "js/views/baseview"],
                 modalType: 'generic',
                 modalSize: 'lg',
                 title: '',
+                modalFocusWindow: '.modal-window',
```

Comment:
A more descriptive name would be modalWindowClass (it has nothing to do with focus per se).